### ISSUE TITLE
Shovel mode filling and emptying silage not working

### ISSUE THREAD

**Issue by BpMac** [Courseplay/courseplay#879]
Hi great work guys!!!
Shovel mode is not working when I try to empty my silage bunker next to my cows.
The tractor drives the course but there is no action with the bucket it just stays static.
Courseplay v4.01.0005
Map Westbridge _Forest_v2.1
Log: https://gist.github.com/BpMac/1096e44715832ea1ab32

**Comment by ThomasGaertner** [comment]
would you please post your courseplay.xml to gist and tell me the name of the course you used ?
Thanks

**Comment by BpMac** [comment]
Okay here is my coursplay.xml for you.
The name of the course is "cowsilage-trailer".
If you look at -line 1112- I think this is what you want.

https://gist.github.com/BpMac

Thanks again for looking into this.

**Comment by ThomasGaertner** [comment]
hmm tested it and worked fine for me... 
would you be so kind to post the vehicles.xml as well ?
Thanks

**Comment by JakobTischler** [comment]
Also, update to build 0011 or higher. I've added "move shovel to ... position" buttons. Activate debug channel 10, and check each position to see if they are set correctly (and post the log).

**Comment by BpMac** [comment]
Hi guys!
Well it seems to be working now. This is what I did.
Uninstalled FS15 and reinstalled FS15 to patch 1.2
Updated to courseplay v4.01.0012
Only added mods courseplay and Westbridge_Forest_V2_1
Used standard tractor and trail for Giants.

Now it seems to be working just fine.
Will have to install my mods one by one to see which one might have caused the problem or maybe updating courseplay or re-installing FS15 made it work.
Thanks for your help!
Keep up the good work!!
